File: Selenium/Activities/Activity9.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver import ActionChains

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/keyboard-events")
    action = ActionChains(driver)
    driver.maximize_window()
    print(f"Title of the page: {driver.title}")

    textArea = driver.find_element(By.XPATH, "//h1[contains(@class, 'mt-3')]")

    action.send_keys("Selenium Test").perform()

    result = textArea.text
    print(f"Text displayed on the webpage: {result}")

File: Selenium/Activities/Activity9.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity9 {

	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		Actions action = new Actions(driver);

		driver.get("https://training-support.net/webelements/keyboard-events");
		driver.manage().window().maximize();
		System.out.println("Title of the page: " +driver.getTitle());
		WebElement textArea = driver.findElement(By.xpath("//h1[contains(@class, 'mt-3')]"));

		action.sendKeys(textArea, "Selenium Test").build().perform();

		String text = textArea.getText();

		System.out.println("Text displayed on the webpage: " +text);

		driver.close();
	}
}
